client/my-sites/stats/stats-purchase/stats-purchase-commercial.tsx

```diff
@@ -59,7 +67,7 @@ const CommercialPurchase = ( { planValue }: CommercialPurchaseProps ) => {
 			<Button variant="primary">
 				{ translate( 'Get Jetpack Stats for %(value)s per month', {
 					args: {
-						value: planValue,
+						value: `$${ planValue }`, //TODO: apply dynamic currency symbol
```

Comment:
The [`format-currency` package](https://github.com/Automattic/wp-calypso/blob/e56b180f31302d4184ae7a0231ff740c465c654a/packages/format-currency/README.md#L4) would come in handy here 🙂

Reply:
Thanks for that. I localised the currency.